frontend/src/pages/result/OverviewTastyDateItems.tsx

```diff
@@ -0,0 +1,68 @@
+import DisplayMenus from "../../components/general/DisplayMenus";
+import React, {useContext, useEffect, useState} from "react";
+import {TastyDateItem} from "../../models/result/TastyDateItem";
+import Box from '@mui/material/Box';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import {getAllTastyDateItems} from "../../service/tastydate-api-service";
+import {AuthContext} from "../../context/AuthProvider";
+import {Button, Card, CardContent} from "@mui/material";
+import "./OverviewVoting.scss"
+import NavigateNextIcon from '@mui/icons-material/NavigateNext';
+import {useNavigate} from "react-router-dom";
+
+
+export default function OverviewTastyDateItems() {
+
+    const {token}=useContext(AuthContext);
+
+    const [selectedTastyDateId, setSelectedTastyDateId]=useState<string>("");
+    const [tastyDateItems, setTastyDateItems]=useState<TastyDateItem[]>([]);
+
+    const navigate = useNavigate();
+
+    const switchPageToCertainVote = () => {
+        navigate(`/overview/${selectedTastyDateId}`);
+    }
+
+    const getEveryTastyDateId = () => {
```

Comment:
```suggestion
    const getEveryTastyDate = () => {
```

Reply:
OK. The naming was obviously confusing. I changed it.